pwndbg/gdblib/regs.py

```diff
@@ -36,6 +37,23 @@ def gdb_get_register(name: str) -> gdb.Value:
         return frame.read_register(name.upper())
 
 
+@pwndbg.gdblib.proc.OnlyWhenRunning
+def get_privileged_register(name: str) -> int:
```

Comment:
nit: not a fan of the name, in some cases GDB already has "privileged" registers depending on the arch, this makes it seem likes this method needs to be used. `monitor info registers` is QEMU specific. Maybe just `get_qemu_register` or `get_register_from_qemu`?